stan/math/rev/fun/log_diff_exp.hpp

```diff
@@ -17,7 +16,7 @@ class log_diff_exp_vv_vari : public op_vv_vari {
   log_diff_exp_vv_vari(vari* avi, vari* bvi)
       : op_vv_vari(log_diff_exp(avi->val_, bvi->val_), avi, bvi) {}
   void chain() {
-    avi_->adj_ += adj_ * calculate_chain(avi_->val_, val_);
+    avi_->adj_ -= adj_ / expm1(bvi_->val_ - avi_->val_);
```

Comment:
There's no way to use inv_logit here? If so, that's fine, just checking.

Reply:
I don't see how you could because one has the extra 1 term inside and the other outside.

```
1 / expm1(x) = 1 / exp(x - 1)
inv_logit(u) = 1 / (1 + exp(-u))
```

Reply:
Oh good point. I was too lazy to actually think through the math at all.